conan/tools/cmake/cmake.py

```diff
@@ -54,9 +55,10 @@ def __init__(self, conanfile, namespace=None):
         self._conanfile = conanfile
         self._namespace = namespace
 
-        toolchain_file_content = load_toolchain_args(self._conanfile.generators_folder, namespace=self._namespace)
-        self._generator = toolchain_file_content.get("cmake_generator")
-        self._toolchain_file = toolchain_file_content.get("cmake_toolchain_file")
+        cmake_presets = load_cmake_presets(conanfile.generators_folder)
+        self._generator = cmake_presets["configurePresets"][0]["generator"]
+        self._toolchain_file = cmake_presets["configurePresets"][0]["toolchainFile"]
+        self._cacheVariables = cmake_presets["configurePresets"][0]["cacheVariables"]
```

Comment:
```suggestion
        self._cache_variables = cmake_presets["configurePresets"][0]["cacheVariables"]
```

Using camelCase because of copy-paste from json?